db/migrate/20221201054153_create_greetings.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
```

Comment:
- [ ] Great job working on this project. As per project requirements, you are supposed to create a route in `routes.rb`, but I cannot find any there. Kindly fix that.

Check if the API endpoint for returning a random greeting was created:
* Go to routes.rb file, check if a route is created.